### PR TITLE
Make compatible with Node.js v22.11.0

### PR DESCRIPTION
**Pull request by awelzel**

### CODE REVIEW

File: docker/fedora-41-nightly.Dockerfile [hunk 1]
```diff
@@ -13,7 +13,7 @@ RUN dnf install -y \
 # Bust the cache
 ARG STAMP=1729535688
 
-RUN dnf config-manager --add-repo https://download.opensuse.org/repositories/security:zeek/Fedora_39/security:zeek.repo
+RUN dnf config-manager addrepo --from-repofile=https://download.opensuse.org/repositories/security:zeek/Fedora_40/security:zeek.repo
```
Comment (bbannier) [inline docker/fedora-41-nightly.Dockerfile:16]: Using packages for fedora-40 on fedora-41 looks like a typo but is actually deliberate (no upstream packages for that distro yet). Could you add a comment calling that out? Adding support for fedora-41 upstream would also be great, maybe you could reference an issue.

File: docker/fedora-41.Dockerfile [hunk 2]
```diff
@@ -13,7 +13,7 @@ RUN dnf install -y \
 # Bust the cache
 ARG STAMP=1729535688
 
-RUN dnf config-manager --add-repo https://download.opensuse.org/repositories/security:zeek/Fedora_39/security:zeek.repo
+RUN dnf config-manager addrepo --from-repofile=https://download.opensuse.org/repositories/security:zeek/Fedora_40/security:zeek.repo
```
Comment (bbannier) [inline docker/fedora-41.Dockerfile:16]: Ditto.

**Review by bbannier — APPROVED** [review]
I managed to build and test with these changes locally, so LGTM 😬.

That said, I see that the CI job with `docker/debian.Dockerfile` might be the one exercising the minimally supported Node version (16.13.1), and it succeeds with these changes. Do you think it would make sense to spell this version requirement out somewhere, e.g., in the readme?

**Comment by timwoj** [comment]
Yep, these are the same failures I mentioned when I dm'd while we were both out.

**Pull request merged**